frontend/simple/js/state.js

```diff
@@ -166,6 +167,18 @@ const getters = {
       )
     }
   },
+  profilesListForGroup (state, getters) {
+    return groupId => {
+      groupId = groupId || state.currentGroupId
+      return groupId && Object.keys(state[groupId].profiles).reduce(
+        (result, username) => {
+          result.push(getters.memberProfile(username, groupId))
+          return result
+        },
+        []
+      )
+    }
+  },
```

Comment:
This function is virtually identical to `profilesForGroup`, except it returns an array. This may be a violation of DRY therefore. Surely the `v-for` above is possible to write with `profilesForGroup` instead?

The [Vue.js API does support looping over objects](https://vuejs.org/v2/api/#v-for):

```html
<div v-for="(item, index) in items"></div>
<div v-for="(val, key) in object"></div>
<div v-for="(val, key, index) in object"></div>
```

Reply:
oh, I didn't know v-for can do that! it's much simpler that way 👍